src/network_enums.h

```diff
@@ -234,6 +234,10 @@ BEGIN_ENUM( SVC )
 	ENUM_ELEMENT ( SVC_SETLINEALPHA ),					// LINE COMMANDS
 	ENUM_ELEMENT ( SVC_SETLINETEXTURE ),
 	ENUM_ELEMENT ( SVC_SETLINETEXTUREBYID ),
+	ENUM_ELEMENT ( SVC_SETLINETEXTUREOFFSET ),
+	ENUM_ELEMENT ( SVC_SETLINETEXTUREOFFSETBYID ),
+	ENUM_ELEMENT ( SVC_SETLINETEXTURESCALE ),
+	ENUM_ELEMENT ( SVC_SETLINETEXTURESCALEBYID ),
```

Comment:
The `SVC` commands are almost full. Unless the function are very bandwidth critical, new function should be added to `SVC2`. i.e. the extended commands.

Reply:
I can't comment on how often these line specials are used in levels, but I can still add these to `SVC2` if they might be better like that. I want to mention that https://github.com/TorrSamaho/zandronum/pull/7 unifies all the `PrintHudMessage` commands into one, which reduced the total `SVC` commands by 3. If the 4 commands above got added in as `SVC` commands, then it would be like adding only one extra `SVC` command.